modules/tinymce/src/core/test/ts/browser/EditorManagerTest.ts

```diff
@@ -38,9 +38,9 @@ describe('browser.tinymce.core.EditorManagerTest', () => {
         assert.equal(EditorManager.get(0), EditorManager.activeEditor);
         assert.isNull(EditorManager.get(1));
         assert.isNull(EditorManager.get('noid'));
-        assert.isNull(EditorManager.get(undefined));
+        assert.isNull(EditorManager.get(undefined as any));
         assert.equal(EditorManager.get()[0], EditorManager.activeEditor);
-        assert.equal(EditorManager.get(EditorManager.activeEditor.id), EditorManager.activeEditor);
+        assert.equal(EditorManager.get((EditorManager.activeEditor as Editor).id), EditorManager.activeEditor);
```

Comment:
Not sure why we need to assert the type of `EditorManager.activeEditor` as Editor when it already is.

Reply:
Take another look 😉 it's `Editor | null` since if there's nothing activated it can be null...